File: examples/user/update/node/updateAnAccount.js
const { UserService } = require("micro-js-client/user");

// Update the account username or email
async function updateAnAccount() {
  let userService = new UserService(process.env.MICRO_API_TOKEN);
  let rsp = await userService.update({
    email: "joe+2@example.com",
    id: "user-1",
  });
  console.log(rsp);
}

updateAnAccount();

File: examples/user/update/go/updateAnAccount.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/user"
)

// Update the account username or email
func UpdateAnAccount() {
	userService := user.NewUserService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := userService.Update(&user.UpdateRequest{
		Email: "joeotheremail@example.com",
		Id:    "usrid-1",
	})
	fmt.Println(rsp, err)
}
